src/ServicePulse.Host/app/js/services/factory.shareddata.js

```diff
@@ -54,6 +63,10 @@
             }
         });
 
+        serviceControlService.getLicenseDetails().then(function (response) {
+            license = response || license;
```

Comment:
Is the `||` here in case the response fails?

Reply:
Yes, I suppose. It's not something I would have considered myself but the call to `getConfiguration` right under it uses the same thing so I figured it must have come up in the past.